src/value.cc

```diff
@@ -238,6 +238,56 @@ Local<Value> GErrorToV8 (GITypeInfo *type_info, GError *err) {
     return obj;
 }
 
+template<typename T>
+static bool isZero(void* value) {
+    return *(reinterpret_cast<T*>(value)) == 0;
+}
+
+static bool isZero(GIArgument &value, GITypeInfo *type_info)
+{
+    GITypeTag type_tag = g_type_info_get_tag (type_info);
+
+    switch (type_tag) {
+        case GI_TYPE_TAG_BOOLEAN:
+            WARN("Boolean zero-terminated array not supported");
+            return false;
```

Comment:
Wouldn't the correct option be to abort here? I'd rather have strict error handling than to allow behaviors that can cause subtle problems later.

Reply:
Well, I took the behavior from Gjs [1]. In this case the resulting array would be an empty array, Anyway, changed to `ERROR()` now.

[1] Don't worry, Gjs is licensed under MIT.